BeeSwiftIntents/IntentHandler.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  IntentHandler.swift
+//  BeeSwiftIntents
+//
+//  Created by Theo Spears on 9/1/22.
+//  Copyright © 2022 APB. All rights reserved.
```

Comment:
Is this still accurate? Code you type is Copyright _andrewpbrett_ ?

Reply:
I think my preference would be to strip all of these comment headers - copyright on opensource files with multiple contributors ends up pretty complex, and all the other info in the headers is redundant.